android-appcoins-billing/src/main/java/com/appcoins/sdk/billing/payasguest/PaymentMethodsFragment.java

```diff
@@ -86,6 +90,7 @@ public static PaymentMethodsFragment newInstance(BuyItemProperties buyItemProper
     super.onCreate(savedInstanceState);
 
     translations = TranslationsRepository.getInstance(getActivity());
+    Activity activity = getActivity();
```

Comment:
you only need the context, why don't you get it from attach method which is safer?